src/components/HeaderBreadcrumbs/HeaderBreadcrumbs.tsx

```diff
@@ -11,22 +11,34 @@ import './HeaderBreadcrumbs.scss';
 const b = block('header-breadcrumbs');
 
 export default function HeaderBreadcrumbs(props: HeaderBreadCrumbsProps) {
-    const {items, theme = 'light', className, analyticsEvents} = props;
+    const {items, theme = 'light', className, analyticsEvents, currentPageUrl} = props;
```

Comment:
As I understand it, this prop will need to be manually registered in the page configuration, I'm not sure that someone will do this every time in order to get aria-current. We need some kind of solution out of the box so that the component understands what the URL is now and does your check. Also, we must not forget that our components are used in ssr